doc/modules/cassandra/pages/architecture/storage_engine.adoc

```diff
@@ -221,5 +224,5 @@ match the "ib" SSTable version
 
 [source,bash]
 ----
-include:example$find_sstables.sh[]
+include::../../examples/BASH/find_sstables.sh[]
```

Comment:
```suggestion
include::example$BASH/find_sstables.sh[]
```

Reply:
Never use a path in includes in asciidoc/antora. https://docs.antora.org/antora/latest/page/include-an-example/